stripe-core/src/main/java/com/stripe/android/core/injection/InitProvider.kt

```diff
@@ -0,0 +1,42 @@
+package com.stripe.android.core.injection
+
+import android.app.Application
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.database.Cursor
+import android.net.Uri
+
+/**
+ * Base class for [ContentProvider]s used for initialization purposes.
+ *
+ */
+abstract class InitProvider : ContentProvider() {
```

Comment:
This just simplifies the usual `ContentProvider` structure by not forcing us to override all methods, as we just care about the `create` function.